x/ccv/provider/types/keys.go

```diff
@@ -564,6 +568,12 @@ func ConsumerRewardsAllocationKey(chainID string) []byte {
 	return append([]byte{ConsumerRewardsAllocationBytePrefix}, []byte(chainID)...)
 }
 
+// ConsumerCommissionRateKey returns the key of consumer chain `chainID` and validator with `providerAddr`
+func ConsumerCommissionRateKey(chainID string, providerAddr ProviderConsAddress) []byte {
+	prefix := ChainIdWithLenKey(ConsumerCommissionRatePrefix, chainID)
```

Comment:
Could we just use `ChainIdAndConsAddrKey(ConsumerComissionRatePrefix, chainID, providerAddr.ToSdkConsAddr())`}

Reply:
Nice catch!